blocks/callout/src/types.ts

```diff
@@ -0,0 +1,43 @@
+import { Entity } from "@blockprotocol/graph";
```

Comment:
[lint](https://github.com/hashintel/hash/pull/2120/commits/59da1019389eefb2099a2095bdd6d17fa9557dee) 

Are these files being linted by eslint?

We might want to disable that, it _does_ say don't edit it 😅  and we probably want to add a CI check later to ensure there isn't a diff

Reply:
Discussed offline – will fix in a follow up https://hashintel.slack.com/archives/C02LG39FJAU/p1677648880232249